repos/fdbt-site/src/pages/api/service.ts

```diff
@@ -26,7 +26,10 @@ export default (req: NextApiRequestWithSession, res: NextApiResponse): void => {
 
         const fareTypeAttribute = getSessionAttribute(req, FARE_TYPE_ATTRIBUTE);
 
-        if (isFareType(fareTypeAttribute) && fareTypeAttribute.fareType === 'return') {
+        if (
+            (isFareType(fareTypeAttribute) && fareTypeAttribute.fareType === 'return') ||
+            (isFareType(fareTypeAttribute) && fareTypeAttribute.fareType === 'period')
```

Comment:
Do we want to do this for all periods or just `return`?

Reply:
@RobCatton so instead of just a `A OR B` should it be A OR (A & B)?

Reply:
No this is correct as is - previously it only needed to go to the returnDirection page if the fareType is return, and now it needs to go there if its period also.

Reply:
Is that true for other period types, such as geoZone and multiService?

Reply:
@RobCatton